velox/functions/remote/if/GetSerde.cpp

```diff
@@ -20,6 +20,8 @@
 
 namespace facebook::velox::functions {
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wreturn-type"
 std::unique_ptr<VectorSerde> getSerde(const remote::PageFormat& format) {
```

Comment:
does gcc really warn about this function? Seems like a trivial example that always returns something...

Reply:
Yes, seems to be a [longstanding issue](https://stackoverflow.com/a/52104273): https://godbolt.org/z/9cKGvdzM1

Reply:
A common practice it to 

```
   return nullptr; // Make compiler happy
```

Reply:
Returning a default VELOX_UNSUPPORTED.